gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -386,7 +409,7 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
     YAML::Node root_node = YAML::LoadFile(impl_->param_file_);
```

Comment:
At the moment the param file is defined as optional but in reality is mandatory.
If it is not provided the plugin crashes here.

For our use case it's better to load the controllers afterward, so we can reuse the same infrastructure for loading them in gazebo, other simulators and on the real robot.

Reply:
This is a bit more complex than I thought because yaml files cannot be easily loaded into a running Node in ROS2.

The only way I've found is to load as part of a `Node` action in a launch file. If the Node is running, there's no `ros2 param load` or equivalent launch action to load a yaml into the param server.  https://github.com/ros2/ros2cli/issues/589

If I have time I'll try to add the ros2 param load, but I can't guarantee.